UIViewController+Extensions.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIViewController+Extensions.swift
```

Comment:
please move this into the `/URBNSwifyConvenience/Classes/` folder and re-add it to the project & `pod install`. I was unable to run the Example without doing this; thats where all the classes should be